apps/heft/src/utilities/RushUtilities.ts

```diff
@@ -38,6 +38,11 @@ export function getRushCommonTempFolder(): string {
   return path.join(rushJsonFolder, 'common', 'temp');
 }
 
+export function getRushCommonTempSplitFolder(): string {
+  const rushJsonFolder: string = findRushJsonFolder();
+  return path.join(rushJsonFolder, 'common', 'temp-split');
```

Comment:
Although my prototype called this folder `common/temp-split`, ideally we'd want to move this under the `common/temp` folder, so that `rush purge` deletes both of them.

However right now Rush generates its workspace as `common/temp/package.json` and `common/temp/pnpm-workspace.yaml`.  So for example if we called the new folder `common/temp/split` then `common/temp/node_modules` would become a phantom dependency parent folder.

🤔 Long term, maybe the best thing would be for Rush to move its workspace into a subfolder such as `common/temp/workspace`.  @D4N14L @dmichon-msft @elliot-nelson Do you have any tools with hardwired paths for `common/temp/package.json`?  Would it be disruptive to move it?

(@chengcyber For this PR, we can simply update `rush purge` to delete `temp-split` which it looks like you already did. 👍)

Reply:
From HBO side we actually do have a couple side scripts that read the pnpm lock file, but none that couldn't easily be updated during a rush upgrade.